src/components/BulkEnrollmentPage/stepper/AddLearnersStep.jsx

```diff
@@ -116,6 +116,8 @@ const AddLearnersStep = ({
     getRowId: (row, relativeIndex, parent) => row?.uuid || (parent ? [parent.id, relativeIndex].join('.') : relativeIndex),
   }), []);
 
+  const filterStatusComponent = () => <DataTable.FilterStatus showFilteredFields={false} />;
```

Comment:
In general, I would move this to the top of the file with the other const components and follow the naming convention in the file already: since it returns a component, I think "const filterStatusComponent" should be "const FilterStatus"